ui/app/templates/vault/cluster/secrets/backend/overview.hbs

```diff
@@ -32,26 +32,31 @@
   <div class="box is-fullwidth is-shadowless has-tall-padding">
     <div class="selectable-card-container {{if (and (eq this.model.connections 403) (eq this.model.roles 403)) 'one-card'}}">
       {{#if this.model.connectionCapabilities.canList}}
-        <SelectableCard
+        <OverviewCard
```

Comment:
Thanks for updating this to use `OverviewCard` instead!

Reply:
I know we desperately need a component library but hopefully this cleans things up for our future selves!